docs/excel/custom-functions-debugging.md

```diff
@@ -43,10 +42,10 @@ You can use the browser developer tools to debug custom functions in Excel Onlin
 ### Start debugging
 
 1. Open developer tools in the browser. For Chrome and most browsers F12 will open the developer tools.
-2. In developer tools, open your source code script file (functions.js or functions.ts). Your custom functions code may be located near the end of the file.
-3. In the custom function source code, apply a breakpoint by selecting a line of code.
+2. In developer tools, open your source code script file (functions.js or functions.ts).
+3. In the custom function source code, select the margin next to the code line number to set a breakpoint on that line.
```

Comment:
"select" implies selecting text. The VS Code does mention to "click in the margin" to set a breakpoint, so I used that terminology.   

"Breakpoints can be toggled by clicking on the editor margin."
https://code.visualstudio.com/docs/editor/debugging#_breakpoints

Reply:
We can't use click because it's not accessible, so MSFT voice guidelines says to use select.

Reply:
Can we just say to "set a breakpoint" and then refer to VS Code docs for more info?

Reply:
```suggestion
3. In the custom function source code, you can toggle a breakpoint in the margin of the code. For more information on setting breakpoints and advanced breakpoint settings, see [Debugging in Visual Studio](https://code.visualstudio.com/Docs/editor/debugging#_breakpoints). 
```
Something like this? Ideally, we don't really want them to have to read another article to figure it out but it's worth putting in the first mention I think in case someone is new

Reply:
No, more like:

3. [Set a breakpoint](https://code.visualstudio.com/Docs/editor/debugging#_breakpoints) in the custom function source code.

Reply:
Developers will know how to set a breakpoint. We don't have to be overly explanatory about it.

Reply:
Ah ok!

Reply:
```suggestion
3. [Set a breakpoint](https://code.visualstudio.com/Docs/editor/debugging#_breakpoints) in the custom function source code. 
```
FWIW I put in a PR to the VS Code guys' docs because they shouldn't use click either